src/conventional/amdar_bufr2ioda.py

```diff
@@ -0,0 +1,506 @@
+#!/usr/bin/python
+
+from datetime import datetime, timedelta
+import dateutil.parser
+import os
+from pathlib import Path
+import sys
+import time
+import logging
+
+import numpy as np
+import netCDF4 as nc
+from eccodes import *
```

Comment:
this (`import *`) is usually very frowned upon because then you don't know where the functions/classes/etc. are coming from.

Reply:
Should this be solely ```import eccodes``` with nothing after it?  I based mine on Ben Ruston's python script for raob.

Reply:
what actually are you using from `eccodes`? The problem with `import *` is it just makes everything available in this space and can overwrite things. After doing this, then you need to change whatever is used in eccodes to be `eccodes.thing` later in this script

Reply:
I believe that @CoryMartin-NOAA is suggesting to use something like this:
```suggestion
from eccodes import CODES_MISSING_LONG, codes_bufr_new_from_file, codes_set, codes_get_array, KeyValueNotFoundError, codes_release
```